src/detect-engine-threshold.c

```diff
@@ -82,7 +82,7 @@ void ThresholdDestroy(void)
 typedef struct ThresholdEntry_ {
     uint32_t key[5];
 
-    uint32_t tv_timeout;    /**< Timeout for new_action (for rate_filter)
+    uint64_t tv_timeout;    /**< Timeout for new_action (for rate_filter)
```

Comment:
@jlucovsky do you have better ideas for these u32 times ?

Reply:
SCTime_t is also 64bit, but then with precision